mopidy_pummeluff/actions/playback.py

```diff
@@ -6,7 +6,7 @@
     'PlayPause',
     'Stop',
     'PreviousTrack',
-    'NextTrack',
+    'NextTrack'
```

Comment:
@hayribakici can you leave the trailing comma there please? :) See https://development-guidelines.confirm.ch/guidelines/python.html:

> If you’re expanding both lists and you make a diff, you’ll notice that the changes of spam are much harder to tell than the changes of eggs. Therefor prefer the multi- over the single-line format. Also make sure you always add a trailing comma to the last element as well.

Reply:
okay, 
it's just, that the linter was unhappy because of it, afaik.